asmcomp/reloadgen.ml

```diff
@@ -113,7 +113,7 @@ method private reload i =
       let newarg = self#makeregs i.arg in
       insert_moves i.arg newarg
         (instr_cons (Iswitch(index, Array.map (self#reload) cases)) newarg [||]
-          (self#reload i.next))
+        (self#reload i.next))
```

Comment:
This change in indentation is not warranted and makes the code harder to read.  `(self#reload i.next)` is an argument to `(instr_cons` above.

Reply:
Yes, odd indentation change. Fixed.